<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow{
background-color: rgb(255, 255, 255);
border:2px solid #14213d;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>700</width>
      <height>550</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame{

	background-color:#14213d;
	border:6px solid #FFDB70;

border-radius:15px;

}
</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QFrame" name="framemonth">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>90</y>
       <width>200</width>
       <height>200</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QFrame {
	border:5px solid #a7a7a7;
	border-radius: 100px;
}
QFrame:hover {
	border: 5px solid #ffdb70;
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>19</x>
        <y>40</y>
        <width>161</width>
        <height>25</height>
       </rect>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>25</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Gandom</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: none;
color:white;
border: none;
</string>
      </property>
      <property name="text">
       <string>ماه را وارد کنید</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="edtmonth">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>110</y>
        <width>141</width>
        <height>30</height>
       </rect>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>30</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit {
	border: 1px solid white;
	border-radius:10px;
}
QLineEdit:hover {
	border: 3px solid #ffdb70;
}</string>
      </property>
      <property name="inputMask">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>80</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Gandom</family>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border:none;
color: #ffdb70;</string>
      </property>
      <property name="text">
       <string>مثال  12 - 8   </string>
      </property>
     </widget>
    </widget>
    <widget class="QRadioButton" name="radioButtonmiladi">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>320</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Gandom</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::RightToLeft</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>تبدیل به میلادی</string>
     </property>
    </widget>
    <widget class="QLabel" name="labeldate">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>440</y>
       <width>501</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>17</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border:none;
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QFrame" name="frameyear">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>90</y>
       <width>200</width>
       <height>200</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QFrame {
	border:5px solid #a7a7a7;

	border-radius: 100px;
}
QFrame:hover {
	border: 5px solid #ffdb70;
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>19</x>
        <y>40</y>
        <width>161</width>
        <height>25</height>
       </rect>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>25</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Gandom</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: none;
color:white;
border: none;
</string>
      </property>
      <property name="text">
       <string>سال را وارد کنید</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="edtyear">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>110</y>
        <width>141</width>
        <height>30</height>
       </rect>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>30</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit {
	border: 1px solid white;
	border-radius:10px;
}
QLineEdit:hover {
	border: 3px solid #ffdb70;
}</string>
      </property>
      <property name="inputMask">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>80</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Gandom</family>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border:none;
color: #ffdb70;</string>
      </property>
      <property name="text">
       <string>مثال 2020 - 1378   </string>
      </property>
     </widget>
    </widget>
    <widget class="QFrame" name="frameday">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>90</y>
       <width>200</width>
       <height>200</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QFrame {
	border:5px solid #a7a7a7;
	border-radius: 100px;
}
QFrame:hover {
	border: 5px solid #ffdb70;
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>19</x>
        <y>40</y>
        <width>161</width>
        <height>25</height>
       </rect>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>25</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Gandom</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: none;
color:white;
border: none;
</string>
      </property>
      <property name="text">
       <string>روز را وارد کنید</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="edt_day">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>110</y>
        <width>141</width>
        <height>30</height>
       </rect>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>30</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit {
	border: 1px solid white;
	border-radius:10px;
}
QLineEdit:hover {
	border:3px solid #ffdb70;
}</string>
      </property>
      <property name="inputMask">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>80</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Gandom</family>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border:none;
color: #ffdb70;</string>
      </property>
      <property name="text">
       <string>مثال 21 - 5     </string>
      </property>
     </widget>
    </widget>
    <widget class="QRadioButton" name="radioButtonshamsi">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>320</y>
       <width>181</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Gandom</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::RightToLeft</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>تبدیل به شمسی</string>
     </property>
    </widget>
    <widget class="QPushButton" name="convertbtn">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>380</y>
       <width>100</width>
       <height>35</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Gandom</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">
color: rgb(0, 0, 0);
border-radius: 10px;
background-color:qlineargradient(spread:pad, x1:0.006, y1:0.585, x2:1, y2:0, stop:0.369318 rgba(246, 176, 66, 255), stop:1 rgba(249, 237, 78, 255));</string>
     </property>
     <property name="text">
      <string>تبدیل</string>
     </property>
    </widget>
    <widget class="QLabel" name="Title">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>18</y>
       <width>231</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>A Salamat</family>
       <pointsize>9</pointsize>
       <weight>50</weight>
       <bold>false</bold>
       <kerning>true</kerning>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">
color: #a7a7a7;
border:none;</string>
     </property>
     <property name="text">
      <string>Toolbox &lt;span style=&quot; font-size:11pt; font-weight:600;&quot;&gt;-&lt;/span&gt; Date Converter</string>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="ExitButton">
     <property name="geometry">
      <rect>
       <x>655</x>
       <y>20</y>
       <width>25</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>C:/Users/Fateme/Downloads/Icon awesome-power-off.png</normaloff>C:/Users/Fateme/Downloads/Icon awesome-power-off.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>edtyear</tabstop>
  <tabstop>edtmonth</tabstop>
  <tabstop>edt_day</tabstop>
  <tabstop>radioButtonmiladi</tabstop>
  <tabstop>radioButtonshamsi</tabstop>
  <tabstop>convertbtn</tabstop>
  <tabstop>ExitButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
